VirtualLibrary/DataSources/Data/Constants.cs

```diff
@@ -10,9 +10,13 @@ static class Constants
     {
         public const string senderEmail = "nergei87@gmail.com";
```

Comment:
Such configuration things could go into email-details file that is already used for holding the credentials, o ever better - an app settings file